platform-sdk/swirlds-common/src/main/java/com/swirlds/common/threading/manager/ThreadManagerFactory.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.common.threading.manager;
+
+import com.swirlds.common.threading.manager.internal.AdHocThreadManager;
+import com.swirlds.common.threading.manager.internal.StandardThreadManager;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Builds thread managers.
+ */
+public final class ThreadManagerFactory {
+
+    private static final AdHocThreadManager STATIC_THREAD_MANAGER = new AdHocThreadManager();
+
+    private ThreadManagerFactory() {}
+
+    /**
+     * Get a static thread manager.
+     *
+     * @return the static thread manager
+     * @deprecated avoid using the static thread manager wherever possible
+     */
+    @Deprecated
```

Comment:
Can we deprecate for removal here?  This should probably be removed sooner rather than later.